frontend/apps/filemanager/filemanager.lua

```diff
@@ -454,6 +455,25 @@ function FileManager:tapPlus()
                     UIManager:close(self.file_dialog)
                 end
             }
+        },
+        {
+            {
+                text = _("Favorite folders"),
+                callback = function()
+                    local fm_bookmark =  FileManagerBookmark:new{
+                        title = _("Favorite folders"),
```

Comment:
As an aside, I ever so slightly prefer "bookmarks" over "favorites," not sure why. In this case I'm a bit torn as to whether it's clearer to be consistent in the naming or not. It might indeed be clearer to have bookmarks in the reader and favorites in the filemanager to prevent confusion.

But the main reason I'm commenting is that I think it's a bit weird to have a split between the code and the text here. It's not like this is older code in which we changed the display text to favorites instead of bookmarks, so why create a distinction? ;-)

Reply:
I'm also torn between the `favorite folders` and the `bookmark folders` :)
What are your opinions? @poire-z  @cramoisi  @KenMaltby @AlanSP1

Reply:
I also didn't like that IE in the 90s called Favorites what Netscape called bookmarks :)
But in our context, bookmarks apply to content. For folders, _bookmarked folders_ sounds strange. _Favorite folders_ sounds just right.
So, I would just rename filemanagerbookmark.lua to filemanagerfavorites.lua and use FileManagerFavorites:new[...}

Reply:
It's never felt odd to me. Many file managers call it bookmarks.

Thunar calls it shortcuts.

Finder (Mac OS) called it favorites.

Nemo calls it bookmarks:
![screenshot_2018-08-18_14-35-58](https://user-images.githubusercontent.com/202757/44299336-19995800-a2f4-11e8-828f-5235a9c7bb6c.png)

Nautilus calls it bookmarks. (Of course Nemo is a Nautilus fork so they're not completely dissimilar.)

Explorer++ calls it bookmarks.

Directory Opus, my favorite file manager, calls it favorites. But of course it's targeted as an Explorer replacement for Windows, so it doesn't deviate on minor things like that without good reason.

Total Commander calls it a Directory Hotlist.

Reply:
> I also didn't like that IE in the 90s called Favorites what Netscape called bookmarks :)

I just realized why I dislike favorites. They're *not* my favorites. They're just things I use a lot. They are indeed shortcuts or bookmarks.

Reply:
For me, favorite folders sounds more "good" than bookmarked folders.

But, it is just the sound of it, either is just fine with me.

Reply:
I prefer the shortcuts from Thunar.